tsconfig.json

```diff
@@ -19,6 +19,7 @@
             "@libs/*": ["./src/libs/*"],
             "@navigation/*": ["./src/libs/Navigation/*"],
             "@pages/*": ["./src/pages/*"],
+            "@scripts/*": ["./scripts/*", "./.github/scripts/*"],
```

Comment:
This sets us up to share code between `./scripts` and our GitHub Actions.

I don't think it would be a bad idea to just get rid of `.github/scripts/` and move them all into `./scripts`, but decided not to do that here to keep the scope clean.